deploy.ci.acceptance.mariadb.dynamic-store-off.prefer-lowest.yml

```diff
@@ -0,0 +1,177 @@
+version: '0.1'
+
+namespace: spryker_ci
+tag: '1.0'
+
+environment: docker.ci
+pipeline: docker.ci.acceptance.dynamic-store-off
```

Comment:
[Minor] Maybe we could add `.prefer-lowest` postfix, here & elsewhere.

Reply:
I would not do that because then we have to create one more `docker.ci.acceptance.dynamic-store-off.prefer-lowest.yml` in `config/install` which will be entirely similar to `docker.ci.acceptance.dynamic-store-off.yml`